bulk_site_creator/management/commands/update_pending_job_tasks.py

```diff
@@ -0,0 +1,139 @@
+import logging
+
+import boto3
+from boto3.dynamodb.conditions import Key
+from coursemanager.models import CourseInstance
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandParser
+import datetime
+
+logger = logging.getLogger(__name__)
+
+dynamodb = boto3.resource("dynamodb")
+table_name = settings.BULK_COURSE_CREATION.get("site_creator_dynamo_table_name")
+
+
+class Command(BaseCommand):
+	help = ('Updates bulk site creator DynamoDB records with workflow_state set to pending to fail '
+	        'and update CourseInstance bulk_processing flag to false')
+
+	def add_arguments(self, parser: CommandParser) -> None:
+		parser.add_argument('--job-id', type=str, required=True, help='The Job ID (SK for main record PK for tasks) value in Dynamo; eg: JOB#12345')
+		parser.add_argument('--school-id', type=str, required=True, help='The School ID that is the primary job PK value in Dynamo; eg: SCHOOL#COLGSAS')
+
+	def handle(self, *args, **options):
+		job_id = options['job_id']
+		school_id = options['school_id']
+		table = dynamodb.Table(table_name)
+		course_instance_id_list = []
+
+		query_params = {
+			'KeyConditionExpression': Key('pk').eq(job_id),
+			'FilterExpression': 'workflow_state = :state',
+			'ExpressionAttributeValues': {':state': 'pending'}
+		}
+
+		query_items = query_dynamo(query_params)
+
+		for item in query_items:
+			sk = item['sk']
+			course_instance_id_list.append(item['course_instance_id'])
+
+			logger.info(f'Updating task {sk} for job {job_id} with workflow_state from pending to fail')
+			now = datetime.datetime.now()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. The recommended way to create an aware datetime object representing a specific timestamp in UTC is by passing `tzinfo` as an argument to the method.

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)

Similar issue at line number 78.